packages/connectivity/lib/connectivity.dart

```diff
@@ -44,7 +44,8 @@ class Connectivity {
   }
 }
 
-ConnectivityResult _stringToConnectivityResult(String state) {
+ConnectivityResult _stringToConnectivityResult(dynamic event) {
```

Comment:
The method should probably be renamed now.

Reply:
I pulled the dynamic/string conversion into an anonymous function at the call site.